docs/Configuration.md

```diff
@@ -1,48 +1,58 @@
+- **skipListWhenSinglePaymentMethod** - Determines whether to enable skipping payment list step when there is only one non-instant payment method. Defaults to `false`.
 
 ### Card component
-* **showStorePaymentField** - Indicates if the field for storing the card payment method should be displayed in the form. Defaults to true.
-* **holderNameRequired** - Indicates if the field for entering the holder name should be displayed in the form. Defaults to false.
-* **hideCvcStoredCard** - Indicates whether to show the security code field on a stored card payment. Defaults to false.
-* **hideCvc** - Indicates whether to show the security code field at all. Defaults to false.
-* **addressVisibility** - Indicates the display mode of the billing address form. Options: `"none"`, `"postal"`, `"full"`. Defaults to `"none"`.
-* **kcpVisibility** - Indicates whether to show the security fields for South Korea issued cards. Options: `"show"` or `"hide"`. Defaults to `"hide"`.
-* **socialSecurity** - Indicates the visibility mode for the social security number field (CPF/CNPJ) for Brazilian cards. Options: "show" or `"hide"`. Defaults to `"hide"`.
-* **supported** - The list of allowed card types. By default uses list of `brands` from payment method. Fallbacks to list of all known cards.
+
+- **showStorePaymentField** - Indicates if the field for storing the card payment method should be displayed in the form. Defaults to true.
+- **holderNameRequired** - Indicates if the field for entering the holder name should be displayed in the form. Defaults to false.
+- **hideCvcStoredCard** - Indicates whether to show the security code field on a stored card payment. Defaults to false.
+- **hideCvc** - Indicates whether to show the security code field at all. Defaults to false.
+- **addressVisibility** - Indicates the display mode of the billing address form. Options: `"none"`, `"postal"`, `"full"`. Defaults to `"none"`.
+- **kcpVisibility** - Indicates whether to show the security fields for South Korea issued cards. Options: `"show"` or `"hide"`. Defaults to `"hide"`.
+- **socialSecurity** - Indicates the visibility mode for the social security number field (CPF/CNPJ) for Brazilian cards. Options: "show" or `"hide"`. Defaults to `"hide"`.
+- **supported** - The list of allowed card types. By default uses list of `brands` from payment method. Fallbacks to list of all known cards.
 
 ### ApplePay component
-* **merchantID** - The merchant identifier for apple pay.
-* **merchantName** - The merchant name. Used for generation of collection of `PKPaymentSummaryItem`.
-* **allowOnboarding** - The flag to toggle onboarding. If `true`, allow the shopper to add cards to Apple Pay if non exists yet. If `false`, Apple Pay is disabled when the shopper doesn’t have supported cards on Apple Pay wallet. Default is `false`.
+
+- **merchantID** - The merchant identifier for apple pay.
```

Comment:
```suggestion
- **merchantID** - The merchant identifier for Apple Pay.
```

Reply:
Fixed in [7567c07](https://github.com/Adyen/adyen-react-native/pull/232/commits/7567c071a9dbab954f3c478a2550fcc483717ef4)